### DISCUSSION TITLE
To-do List & Report

### DISCUSSION THREAD

**Discussion by suan073** [suan073/NEEDle_ideaBank#11] [Show and tell]
1. 본인이 할일 목록과 각 목록 옆에 언제 까지 할 것인지 작성
2. 당일에 한 일 보고

**Comment by kimmokalover** [comment]
**Date: 2023-01-11 WED**

**Reply by kimmokalover** [reply]
- 승준
- [x] util.py에 파일 초기화 기능 구현
- [x] 데이터 시각화

**Reply by suan073** [reply]
* 수인
- [ ] skip-gram & 네거티브 샘플링 코멘트 작성 → 보류
- [ ] 병렬 학습 수단 조사 후 노션 기록 → 잔업

**Reply by ghost** [reply]
- 태민
- [x] knn 가능성 확인 (가능할 시 구현) -> 불가능
- [ ] knn 모델 구현 -> 노션 기입 X
- [ ] 적절한 하이퍼 파라미터 찾기 X
- [x] 다른 방안 찾기
- [x] 구현

**Comment by kimmokalover** [comment]
Date: 2023-01-13 FRI

**Reply by kimmokalover** [reply]
- 승준
- [ ] 데이터 클러스터링
- [ ] 쿠버네티스 알아보기

**Reply by ghost** [reply]
- 태민
- [x] 인풋 레이어 모델 완성하기

**Comment by suan073** [comment]
Date: 2023-01-15 SUN

**Reply by ghost** [reply]
- 태민
- [x] 인풋 레이어 loss값 똑같은 이유 찾고 수정하기

> - [x] 할 수 있다면, 모델 정교하게 수정하기
- [ ] 쿠버네티스 공부하고 backend 간단하게 짜보기

**Reply by suan073** [reply]
* 수인 
- [x] Needs 구체화시키기

**Comment by kimmokalover** [comment]
Date: 2023-01-16 MON

**Reply by kimmokalover** [reply]
- 승준
- [x] entity diagram
- [x] mysql create table

**Reply by suan073** [reply]
* 수인 
- [x] clustering.ipynb 실행 → https://www.notion.so/clustering-ipynb-a0858f97fbeb4bdd8cb40b542736845a
- [x] 진행 상황 정리 및 일정 잡기 → https://www.notion.so/01-20-ee17406c750e4f828dd7c4e0ebb29ca8

**Comment by suan073** [comment]
Date: 2023-01-17 MON

**Reply by suan073** [reply]
- [ ] 2분할 클러스터링 구현 및 실행 ( AF~ )
- [ ] skip-gram 테스트
- [ ] AWS 인프라 확인
